Dockerfile.windows

```diff
@@ -1,17 +1,28 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+FROM microsoft/dotnet-nightly:2.0-sdk-nanoserver as builder
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
+ENV NUGET_XMLDOC_MODE skip
+ENV DOTNET_SKIP_FIRST_TIME_EXPERIENCE 1
+
 RUN New-Item -Path \MusicStore\samples\MusicStore -Type Directory
-WORKDIR MusicStore
+WORKDIR app
 
 ADD samples/MusicStore/MusicStore.csproj samples/MusicStore/MusicStore.csproj
+ADD build/dependencies.props build/dependencies.props
 ADD NuGet.config .
-RUN dotnet restore .\samples\MusicStore
+RUN dotnet restore --runtime win10-x64 .\samples\MusicStore
 
 ADD samples samples
-RUN dotnet build .\samples\MusicStore
+
+RUN dotnet publish --output /out --configuration Release --framework netcoreapp2.0 --runtime win10-x64 .\samples\MusicStore
+
+FROM microsoft/dotnet-nightly:2.0-runtime-nanoserver
```

Comment:
Can you change this to use `microsoft/aspnetcore-nightly:2` instead? We recently started producing these nightlies.